packages/babel-parser/test/fixtures/flow/optional-type/2/options.json

```diff
@@ -1,3 +1,3 @@
 {
-  "throws": "Unexpected token (1:13)"
+  "throws": "Unexpected token (1:12)"
```

Comment:
The input code is `const f = (x?)`. The old error position was at `)`; now it's at `?`.